### PR TITLE
Redundant links

### PR DESCRIPTION
**Pull request by simonp20**
Fixed WAVE redundant link errors in the col-block-small, col-block, and row-block files by putting the titles for the sites into the <a> tags.

### CODE REVIEW

File: wp-content/themes/bpr2018/resources/templates/partials/post/col-block-small.tpl.php [hunk 1]
```diff
@@ -17,15 +17,14 @@ class="img-10"
...
     </a>
   </div>
 
   <div class="text col-sm-7">
-    <div class="post-title-small">
-      <a itemprop="url" href="<?php echo esc_url(get_permalink()); ?>">
-      <span itemprop="headline"><?php the_title(); ?></span>
-      </a>
-    </div>
+
```
Comment (CaoRuiming) [inline wp-content/themes/bpr2018/resources/templates/partials/post/col-block-small.tpl.php:27]: Could you actually undo the changes to this file? I think this kind of change would actually break things if we were to ever use this template again.

Comment (CaoRuiming) [inline wp-content/themes/bpr2018/resources/templates/partials/post/col-block-small.tpl.php:27]: Could you actually undo the changes to this file? I think this kind of change would actually break things if we were to ever use this template again.

**Review by CaoRuiming — CHANGES_REQUESTED** [review]
One super minor change, but this looks good otherwise.

File: wp-content/themes/bpr2018/resources/templates/partials/post/col-block-small.tpl.php (hunk 1 above)
Comment (simonp20) [reply wp-content/themes/bpr2018/resources/templates/partials/post/col-block-small.tpl.php]: Done!

**Review by CaoRuiming — APPROVED** [review]
